test/Filesystem.test.cpp

```diff
@@ -17,6 +17,16 @@ class FilesystemTest : public ::testing::Test {
 };
 
 
+TEST_F(FilesystemTest, basePath) {
+	// Result is a path, so should end with a directory separator
```

Comment:
Result is a *directory* not a filepath. Not a big deal since this is a comment but should be consistent.

Reply:
Hmm, might have been leftover from when I did a bit of renaming.